sfge/src/main/java/com/salesforce/graph/symbols/apex/ApexStringValue.java

```diff
@@ -273,125 +484,226 @@ public Optional<ApexValue<?>> apply(MethodCallExpressionVertex vertex, SymbolPro
+                    convertParameterToString(parameters.get(0), symbols).orElse(null);
+            final Integer parameter2 =
+                    convertParameterToInteger(parameters.get(1), symbols).orElse(null);
+            if (parameter1 != null && parameter2 != null) {
+                return Optional.of(
+                        builder.buildString(
+                                METHOD_NAME_TO_TRIFUNCTION_STRING_STRING_INTEGER_STRING
+                                        .get(methodName)
+                                        .apply(value, parameter1, parameter2)));
+            }
+        } else if (numParams == 2
+                && METHOD_NAME_TO_TRIFUNCTION_STRING_INTEGER_INTEGER_STRING_METHOD.containsKey(
+                        methodName)) {
+            final Integer parameter1 =
+                    convertParameterToInteger(parameters.get(0), symbols).orElse(null);
+            final Integer parameter2 =
+                    convertParameterToInteger(parameters.get(1), symbols).orElse(null);
+            if (parameter1 != null && parameter2 != null) {
+                return Optional.of(
+                        builder.buildString(
+                                METHOD_NAME_TO_TRIFUNCTION_STRING_INTEGER_INTEGER_STRING_METHOD
+                                        .get(methodName)
+                                        .apply(value, parameter1, parameter2)));
             }
-        } else if (METHOD_SUB_STRING.equalsIgnoreCase(methodName)) {
-            validateParameterSize(vertex, 2);
-            final Integer from = convertParameterToInteger(parameters.get(0), symbols).orElse(null);
-            final Integer to = convertParameterToInteger(parameters.get(1), symbols).orElse(null);
-            if (from != null & to != null) {
-                return Optional.of(builder.buildString(value.substring(from, to)));
+        } else if (numParams == 2
+                && METHOD_NAME_TO_TRIFUNCTION_STRING_STRING_INTEGER_STRING_LIST_METHOD.containsKey(
+                        methodName)) {
+            final String parameter1 =
+                    convertParameterToString(parameters.get(0), symbols).orElse(null);
+            final Integer parameter2 =
+                    convertParameterToInteger(parameters.get(1), symbols).orElse(null);
+            if (parameter1 != null && parameter2 != null) {
+                ApexListValue apexListValue = builder.deepClone().buildList();
+                List<String> outList =
+                        METHOD_NAME_TO_TRIFUNCTION_STRING_STRING_INTEGER_STRING_LIST_METHOD
+                                .get(methodName)
+                                .apply(value, parameter1, parameter2);
+                for (String result : outList) {
+                    apexListValue.add(builder.deepClone().buildString(result));
+                }
+                return Optional.of(apexListValue);
             }
+        } else if (METHOD_STRIP_HTML_TAGS.equalsIgnoreCase(methodName)) {
```

Comment:
The code for stripHtmlTags is custom and written inside of core (in the eleanor module, see StripHtmlTags.java).
We don't want the graph engine to error here so for now it's easiest to just return an indeterminate value for now.